pages/login.js

```diff
@@ -0,0 +1,27 @@
+import { useSession, signIn, signOut } from "next-auth/react";
+import Button from "@/components/Button";
+import { useRouter } from "next/router";
+export default function Login() {
+  const { data: session } = useSession();
+  const router = useRouter();
+  if (session) {
```

Comment:
that is looking a little unstructured here.

If you use if(), also use the else() later, 
or at all use the short form `ternery`.